config/clusters/2i2c-aws-us/dask-staging.values.yaml

```diff
@@ -0,0 +1,59 @@
+basehub:
+  userServiceAccount:
+    annotations:
+      eks.amazonaws.com/role-arn: arn:aws:iam::790657130469:role/2i2c-aws-us-dask-staging
+  nfs:
+    pv:
+      # from https://docs.aws.amazon.com/efs/latest/ug/mounting-fs-nfs-mount-settings.html
+      mountOptions:
+        - rsize=1048576
+        - wsize=1048576
+        - timeo=600
+        - soft # We pick soft over hard, so NFS lockups don't lead to hung processes
+        - retrans=2
+        - noresvport
+      serverIP: fs-0b70db2b65209a77d.efs.us-west-2.amazonaws.com
+      baseShareName: /
+  jupyterhub:
+    custom:
+      2i2c:
+        add_staff_user_ids_to_admin_users: true
+        add_staff_user_ids_of_type: "github"
+      homepage:
+        templateVars:
+          org:
+            name: 2i2c Dask Staging
+            url: https://2i2c.org
+            logo_url: https://2i2c.org/media/logo.png
+          designed_by:
+            name: 2i2c
+            url: https://2i2c.org
+          operated_by:
+            name: 2i2c
+            url: https://2i2c.org
+          funded_by:
+            name: 2i2c
+            url: https://2i2c.org
+    singleuser:
+      image:
+        name: pangeo/pangeo-notebook
+        tag: "2022.06.02"
+    hub:
+      config:
+        Authenticator:
+          # This hub uses GitHub Org auth and so we don't set
+          # allowed_users in order to not deny access to valid members of
+          # the listed orgs.
+          #
+          # You must always set admin_users, even if it is an empty list,
+          # otherwise `add_staff_user_ids_to_admin_users: true` will fail
+          # silently and no staff members will have admin access.
+          admin_users: []
+        JupyterHub:
+          authenticator_class: "github"
+        GitHubOAuthenticator:
+          oauth_callback_url: "https://dask-staging.aws.2i2c.cloud/hub/oauth_callback"
+          allowed_organizations:
+            - 2i2c-org
+          scope:
+            - read:org
```

Comment:
Nice! I think using the GitHubOAuthenticator directly is more straightforward than cilogon + GitHub 👍🏼 
I feel like cilogon is really valuable with institutional providers or when we use to provide multiple providers options

Reply:
Yeah, I agree!